src/components/InactivityTracker/InactivityTracker.js

```diff
@@ -33,8 +34,6 @@ export const InactivityTracker = ({ openedFacilityTabs, setCheckout }) => {
       console.log("Time is up!");
       resetUserInactivityTimer();
 
-      if (handler !== undefined) handler();
-
       logOut(undefined);
```

Comment:
passing 'undefined' is not needed.  not passing anything is the same as passing 'undefined' logically.  a "default" parameter should be used in the function definition, if needed.

Example:

  const logOut = (someArgument = "defaultValue") => {
    // code here
  }